Cargo.toml

```diff
@@ -12,6 +11,7 @@ members = [
   "crates/astria-sequencer-utils",
   "crates/astria-test-utils",
 ]
+resolver = "2"
```

Comment:
Should we set `edition = 2021` instead? https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html

Reply:
See further down in that link:

> The resolver is a global setting for a workspace, and the setting is ignored in dependencies. The setting is only honored for the top-level package of the workspace. If you are using a virtual workspace, you will still need to explicitly set the resolver field in the [workspace] definition if you want to opt-in to the new resolver.